src/AppInstallerCLITests/TestHandler.h

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#pragma once
+#include "pch.h"
+#include "cpprest/http_client.h"
+
+class TestHandler : public web::http::http_pipeline_stage

```

Comment:
>TestHandler [](start = 6, length = 11)

nit:the TestHandler name seems too generic, suggest TestRestRequestHandler to be more specific